internal/cephfs/store/fsjournal.go

```diff
@@ -40,6 +40,10 @@ var (
 	// SnapJournal is used to maintain RADOS based journals for CO generated.
 	// SnapshotName to backing CephFS subvolumes.
 	SnapJournal *journal.Config
+
+	// VolumeGroupJournal is used to maintain RADOS based journals for CO
+	// generated.
+	VolumeGroupJournal journal.VolumeGroupJournal
```

Comment:
```suggestion
	// generated request name to CephFS snapshot group attributes.
	VolumeGroupJournal journal.VolumeGroupJournal
```

Just ending with ` for CO generated.` seems incomplete.